### ISSUE TITLE
This isn't an issue, but just an appreciation post for Kalaborative.

### ISSUE THREAD

**Issue by reflexin** [Kalaborative/survivio-plus#88]
Hey Kalab, I don't think you hear this enough, but I strongly appreciate you and what you're doing. You are under no obligation to fix this for your users but you take time out of your day to do it anyway. It'd appear to be impossible to get everything done alone here, but you still keep it working. You're a great dev and I really appreciate your work!

**Comment by Kalaborative** [comment]
@reflexin Thanks! I just like to help 😄 

@RuanTM Yes, try changing the settings for auto aim prediction or auto aim speed

**Comment by RuanTM** [comment]
yes but I raise the two or below the two or another something specific?
